client/app-client-java/src/main/proto/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015, Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Why is there a proto folder inside the client? Your client should pick up the protos from the defs folder. Please delete this folder and use the protos from the defs folder which  has all the definitions already